recipe/meta.yaml

```
@@ -11,35 +11,41 @@ source:
   - url: https://github.com/robotframework/robotframework/archive/v{{ version }}.tar.gz
     sha256: 1045dc4482f16737f58686b659f2cd8a91750ecb1707389051ada075f79e9e32
     folder: src
+    patches:
+      - patches/0001_skip-tests-wrong-timezone-set.patch
+      - patches/0002_skip-test_clear_namespace_between_runs-for-win.patch  # [win]
 
 build:
-  number: 1
-  noarch: python
+  number: 0
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - wheel
```

```suggestion
    - wheel
    - setuptools
```
(same in the outputs)